### DISCUSSION TITLE
Grafana FIPS 140-2 Certification

### DISCUSSION THREAD

**Discussion by sujaladi** [grafana/grafana#68588] [General]
Hi,
Is Grafana FIPS 140-2 certified? Has anyone used Grafana in their FEDRAMP environments? I haven't found much information about this online. 

Thanks.

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). This is phrased as a question, and we'd like questions about grafana to go to our[ community forum](https://community.grafana.com/). Feel free to make an [issue](https://github.com/grafana/grafana/issues) if you cannot find a solution and would like to request the feature.
